### ISSUE TITLE
Build error because MAX_FACE_NODES<16

### ISSUE THREAD

**Issue by sytuannguyen** [GEOSX/GEOS#2362]
`**Describe` the bug**
Build error because the condition `numFaceNodes<=MAX_FACE_NODES` is not satisfied. The error says that `numFaceNodes=16` and `MAX_FACE_NODES=11` . There is a `GEOSX_ERROR_IF_GT_MSG` to prevent this issue but it seems not working during compile time, only during the run time. 

**Screenshots**
![image](https://user-images.githubusercontent.com/45685596/228188842-1010de6f-c648-4e30-ba8b-9d23ab8c2fb7.png)

**Platform (please complete the following information):**
- Machine [TotalEnergies/XRAI] 
- Compiler: [e.g. gcc 12.1.0]
- GEOSX Version [e.g. 0.2]

**Additional context**
This issue can be avoided by increasing the `MAX_FACE_NODES` to 16.

**Comment by untereiner** [comment]
I personally do not have this bug on xray compiling the develop branch. Is it on a specific commit ?

**Comment by sytuannguyen** [comment]
> I personally do not have this bug on xray compiling the develop branch. Is it on a specific commit ?

@untereiner Not for a spedific commit, I got this bug when compiling the develop branch. The environment source might be different.

**Comment by sytuannguyen** [comment]
@TotoGaz Yes I confirm that the compiler is 12.1.0. I tried compiling different commits of the develop branch and I observed that this issue occurs for all the commits including the one that pass before. I think this is purely related to the environment change within xrai.

**Comment by untereiner** [comment]
@sytuannguyen Is your problem solved ? Did you try with an older version of the compiler ? (e.g: 8.2 on xrai ?)
